modules/channel-teams/src/backend/client.ts

```diff
@@ -64,6 +64,15 @@ If you have a restricted app, you may need to specify the tenantId also.`
       const { activity } = turnContext
 
       const conversationReference = TurnContext.getConversationReference(activity)
+
+      // A new user is reaching the bot for the first time
+      if (activity.type === 'conversationUpdate' && activity.membersAdded?.length) {
+        // Locale format: {lang}-{subtag1}-{subtag2}-... https://en.wikipedia.org/wiki/IETF_language_tag
+        // TODO: Use Intl.Locale().language once its types are part of TS. See: https://github.com/microsoft/TypeScript/issues/37326
+        const lang = activity.locale && activity.locale.split('-')[0]
+        return this._sendProactiveMessage(conversationReference, lang)
+      }
```

Comment:
I'm wondering, can the bot be in a channel with more than one user ? Wouldn't this logic send a message to every user, every time a new user is added in the "group" ? 

Also, you can use conditional chaining `activity.local?.split`

Reply:
> I'm wondering, can the bot be in a channel with more than one user ? Wouldn't this logic send a message to every user, every time a new user is added in the "group" ?

Good catch! I think you are right about this!